tests/databases/api_tests.py

```diff
@@ -801,3 +803,26 @@ def test_get_database_related_objects_not_found(self):
         uri = f"api/v1/database/{database.id}/related_objects/"
         rv = self.client.get(uri)
         self.assertEqual(rv.status_code, 404)
+
+    def test_export_database(self):
+        """
+        Database API: Test export database
+        """
+        self.login(username="admin")
+        uri = "api/v1/database/1/export/"
+        rv = self.client.get(uri)
+
+        self.assertEqual(rv.status_code, 200)
+
+        buf = BytesIO(rv.data)
+        self.assertTrue(is_zipfile(buf))
+
+    def test_export_database_not_allowed(self):
+        """
+        Database API: Test export database
```

Comment:
nit: `Database API: Test export database not allowed`

Reply:
Good catch, thanks!